.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # Necessary to update action hashs	
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    # Allow up to 3 opened pull requests for github-actions versions
+    open-pull-requests-limit: 3
```

Comment:
Do you have an example of such PR?

Reply:
Here's one that's also using dependabot to update a pinned-by-hash github action: https://github.com/OpenKneeboard/OpenKneeboard/pull/353